zephyr-apps/gpio/README.rst

```diff
@@ -0,0 +1,95 @@
+.. _blinky-sample:
+
```

Comment:
Since this is our own sample, I would removed all this, made just add a simple outline of what the project does and change the file to a `README.md` just to be consistent with the other readme's